### PR TITLE
Adding pulp-2to3-migration support

### LINKED ISSUE #133: Feature: Pulp 3 migration plugin support
It would be nice to be able to browse migration data.

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #133

### CODE REVIEW

File: pulpcore/cli/migration/pulp2.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+def pulp2() -> None:
+    pass
+
+
+@pulp2.group()
+@pass_pulp_context
+@click.pass_context
+def content(ctx: click.Context, pulp_ctx: PulpContext) -> None:
+    ctx.obj = PulpMigrationPulp2ContentContext(pulp_ctx)
+    pass
```
Comment (mdellweg) [inline pulpcore/cli/migration/pulp2.py:21]:
```suggestion
```

File: pulpcore/cli/migration/plan.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+)
+from pulpcore.cli.migration.context import PulpMigrationPlanContext
+
+
+@click.group()
+@pass_pulp_context
+@click.pass_context
+def plan(ctx: click.Context, pulp_ctx: PulpContext) -> None:
+    ctx.obj = PulpMigrationPlanContext(pulp_ctx)
+    pass
```
Comment (mdellweg) [inline pulpcore/cli/migration/plan.py:19]:
```suggestion
```

File: tests/scripts/pulp_2to3_migration/test_plan.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(dirname "$(realpath "$0")")")"/config.source
+
+pulp debug has-plugin --name "pulp_2to3_migration" || exit 3
```
Comment (mdellweg) [inline tests/scripts/pulp_2to3_migration/test_plan.sh:6]: Am i right, that these tests will not run in our container based CI?
Would this be reason to have this plugin packaged independently?

Comment (daviddavis) [reply tests/scripts/pulp_2to3_migration/test_plan.sh:6]: > Am i right, that these tests will not run in our container based CI?

That's correct

> Would this be reason to have this plugin packaged independently?

That's a good question. Let's discuss at our CLI meeting?

**Comment by daviddavis** [comment]
Looks like the tests are running successfully against the migration plugin: https://github.com/pulp/pulp-2to3-migration/pull/315

**Review by mdellweg — APPROVED** [review]
Thanks for keeping on top of the tests.

**Pull request merged**